### PR TITLE
Pop up feature

### PR DESCRIPTION
**Pull request by Mahdi-Niazi**
In this milestone I:

- Add desktop pop up functionality
- Add mobile pop up functionality

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -21,4 +22,189 @@ document.querySelectorAll('.nav-item').forEach(n =>
...
+                        <p class="pop-txt"></p>
+                        <div class="btn-div">
+                            <a class="btn-orange pop-btn" id="live" href="">See Live  <i class="fa-solid fa-globe"></i></a>
+                            <a class="btn-orange pop-btn" id="source" href="">See Source <i class="fa-brands fa-github"></i></a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    `;
```
Comment (V-Blaze) [inline script.js:182]: - [x] You have done great so far :+1: :+1: But there are some changes you need to make before moving to the next milestone. :+1: 
- [ ] Currently, on a bigger screen above `1410 px`, your pop-up tends to not fit into the screen. Looking at the screenshot below, you can see that some parts are not visible. I'll suggest you adjust the height and with of this appropriately. :+1: 

- -  Screenshot:
Screenshot from 2023-06-21 18-42-29

- [x] Also on mobile screens, the content is squashed together, and there are no spacing and margins like the Figme design, I'll suggest you look into this too.

- - Your Current design:
Screenshot from 2023-06-21 18-46-51

- -Figma design:
Screenshot from 2023-06-21 18-47-20

File: script.js [hunk 2]
```diff
@@ -21,4 +22,189 @@ document.querySelectorAll('.nav-item').forEach(n =>
...
+
+
+const overlay = document.querySelector('.overlay');
+
+function popupshow() {
+    popup.classList.add('active');
+    overlay.classList.add('active');
+}
+
```
Comment (V-Blaze) [inline script.js:191]: - [x] I  noticed that your popup has just one image for all the projects. That should not be so. The image on the popup should be generated randomly based on the project that was clicked. the image on the project should be the one on the popup. I'll suggest you fix this too. :+1: :smile:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Mahdi-Niazi  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Project section is generated randomly✔
-  Correct GitHub flow👋 
-  Well organized code✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -617,6 +617,105 @@ textarea::placeholder {
...
+  padding: 24px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  z-index: 2;
+  background-color: #fff;
+  box-sizing: border-box;
+  border: 1px solid red;
+}
```
Comment (Strangeal) [inline style.css]: Great job on implementing the changes as specified by the previous reviewer! 👏 However, there is a minor UI issue 🐞 that we need to address as part of the changes suggested by the previous reviewer: 
- [ ] On mobile screens, the content is squashed together, and due to that the contents are overflowing also there is no spacing and margins like that of the `Figma` design. To fix this issue kindly consider below suggestions:
   - [ ]  Increase the set `width` on mobile screens.
   - [ ] Add some spacing between your buttons.
   - [ ] Reduce the height of the pop-up image.
<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td></td>
<td>
Screenshot 2023-06-21 at 21 38 16
</td>
</tr>
</table>

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @Mahdi-Niazi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -617,6 +617,106 @@ textarea::placeholder {
...
+
+.pop-txt {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 30px;
+}
+
```
Comment (usorfaitheloho) [inline style.css:698]: You did a great job with the mobile version of your popup modal, However, as suggested by the previous reviewer about your mobile popup, there is still a little adjustment to be made. Your popup screen doesn't have the full view of your project (that is, some part of the view is truncated)as it is in the Figma template you were given. kindly use the right properties so it looks close to the design as much as possible 😄 

Your design :
image

Figma design :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
Hi @Mahdi-Niazi  :thumbsup:  ,

This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are still some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good use of JavaScript, HTML&CSS best practices ✔️ 
-  Well-structured files ✔️
<br/>
<br/>
<br/>

 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (Mohammad-m-Niazi) [reply style.css:698]: Kindly provide me a better screenshot the problem to understand it better. Thank you

Comment (Mohammad-m-Niazi) [reply style.css:698]: @usorfaitheloho Kindly provide me a better screenshot the problem to understand it better. Thank you

Comment (usorfaitheloho) [reply style.css:698]: @Mahdi-Niazi The screenshot I provided is how your design looks, kindly compare it with the Figma design. I think you should increase the height of your modal popup so it captures your full design 😄 

image

**Review by Cabraham1 — APPROVED** [review]
Hi @Mahdi-Niazi  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 Descriptive pull request✅
📌 Impressive UI ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**